### PR TITLE
Story epic progress

### PR DESCRIPTION
**Pull request by duggiemitchell**
## Cute animal pic
photo-1513223848047-2456e15b4f7d

## Link to Trello card (if applicable)
https://trello.com/c/pjCczuhe

## Description
_The commit messages say what you did; this should explain why and/or how._
- [ ] Description is in Trello card

### CODE REVIEW

**Comment by duggiemitchell** [comment]
[Metecho epics › progress](https://trello.com/c/pjCczuhe/17-metecho-epics-progress)

File: src/stories/epics/progress/index.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Overview
+
+The **Epic Progress** component provides a quick view of the progress when
+viewing a project.
+
+## Guidelines
+
+It accepts a `range` (`[number, number]`) with the first number representing the
+number of tasks in a project with a `Completed` status, and the second
+represents the total number of project tasks.
```
Comment (dvdherron) [inline src/stories/epics/progress/index.stories.mdx:20]:
```suggestion
representing the total number of project tasks.
```

Comment (dvdherron) [inline src/stories/epics/progress/index.stories.mdx:20]:
```suggestion
represents the total number of project **Tasks**.
```

File: src/stories/epics/progress/index.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<Meta title="Epics/Progress/Docs" />
+
+# Epic Progress
+
+**Visualize the overall progress/status of the entire project.**
+
+## Overview
+
+The **Epic Progress** component provides a quick view of the progress when
```
Comment (dvdherron) [inline src/stories/epics/progress/index.stories.mdx:13]: Maybe: "The **Epic Progress** component visualizes the progress made on **Tasks** within an **Epic**." ?

File: src/stories/epics/progress/index.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+viewing a project.
+
+## Guidelines
+
+It accepts a `range` (`[number, number]`) with the first number representing the
+number of tasks in a project with a `Completed` status, and the second
```
Comment (dvdherron) [inline src/stories/epics/progress/index.stories.mdx:19]:
```suggestion
number of  `Completed` **Tasks** in a project, and the second
```

File: src/stories/epics/progress/index.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+represents the total number of project tasks.
+
+<Canvas>
+  <EpicProgress range={[7, 13]} />
+</Canvas>
+
+When all the tasks in a project are complete, we've added a
```
Comment (dvdherron) [inline src/stories/epics/progress/index.stories.mdx:26]:
```suggestion
When all the **Tasks** in a project are complete, a 
```

File: src/stories/epics/progress/index.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+<a href="https://www.lightningdesignsystem.com/components/progress-bar/#Modifiers">color
+modifier </a>to the underlyning progress bar.
```
Comment (dvdherron) [inline src/stories/epics/progress/index.stories.mdx:28]:
```suggestion
modifier </a>changes the progress bar from blue to green.
```

**Review by dvdherron — APPROVED** [review]
Looks good! I just have some minor copy comments/suggestions. 

 And I'm wondering if this line break can be fixed?
image

**Pull request merged**